ivy_tests/test_ivy/test_frontends/test_torch/test_indexing_slicing_joining_mutating_ops.py

```diff
@@ -70,3 +70,38 @@ def test_torch_cat(
         dim=unique_idx,
         out=None,
     )
+
+    
+# reshape
+@given(
+    xs_n_input_dtypes_n_unique_idx=_arrays_idx_n_dtypes(),
```

Comment:
please use `helpers.dtype_and_values` as unique idx is not needed, and set the argument `available_dtypes` to:

```
available_dtypes=tuple(
            set(ivy_np.valid_float_dtypes).intersection(
                set(ivy_torch.valid_float_dtypes)
            )
        )
```